src/spread/mult-iter.case

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+desc: Spread operator following other arguments with a valid iterator
+template: default
+features: [Symbol.iterator]
+info: |
+    12.3.6.1 Runtime Semantics: ArgumentListEvaluation
+
+    ArgumentList : ... AssignmentExpression
+
+    1. Let list be an empty List.
+    2. Let spreadRef be the result of evaluating AssignmentExpression.
+    3. Let spreadObj be GetValue(spreadRef).
+    4. Let iterator be GetIterator(spreadObj).
+    5. ReturnIfAbrupt(iterator).
+    6. Repeat
+       a. Let next be IteratorStep(iterator).
+       b. ReturnIfAbrupt(next).
+       c. If next is false, return list.
+       d. Let nextArg be IteratorValue(next).
+       e. ReturnIfAbrupt(nextArg).
+       f. Append nextArg as the last element of list.
+---*/
+
+//- setup
+var iter = {};
+iter[Symbol.iterator] = function() {
+  var callCount = 3;
+  return {
+    next: function() {
+      callCount += 1;
+      return { done: callCount === 6, value: callCount };
```

Comment:
this local `callCount` is confusing on the generated file that has an assertion [for another callCount](https://github.com/tc39/test262/pull/585/commits/51c3570b5d74bc7c040c2ff410246ae45060aba4#diff-e570bb4dbf156f80980eb89a65bc5074R1). Can we use another name here?